### PR TITLE
builds steps and logs

### PR DESCRIPTION
**Pull request by nicolaifsf**
Builds steps and logs endpoints added

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -125,12 +125,43 @@ func CreateApp() *cli.App {
...
+							return cli.ShowSubcommandHelp(c)
+						}
+						command.FormattedPrint(resp)
+						return nil
+					},
+				},
+				{
+					Name:      "steps-log",
+					Usage:     "Get the logs for a step",
+					ArgsUsage: "<id> <stepName>",
```
Comment (Filbird) [inline client.go:144]: Feels like it's very wordy to get a step log.

e.g.
`sd builds steps-log 1234 step0`

What are your thoughts on simply having a "logs" command that takes `id` and `stepName`?

e.g.
`sd logs 1234 step0`

**Comment by Filbird** [comment]
I'm thinking it may be more conventional if we swapped around the command ordering to something like `sd list pipelines` and `sd get pipeline <id>`.

Do we have preference for one way or the other?

**Comment by stjohnjohnson** [comment]
👍  Awesome

**Comment by nicolaifsf** [comment]
That's a good idea I think Filbert. I'll make the changes.

**Pull request merged**